### PR TITLE
feature/HW-60042 support view model refreshes

### PR DESCRIPTION
**Pull request by peter-joseph**

### CODE REVIEW

File: receiptrepository/src/main/java/com/hyperwallet/android/ui/receipt/repository/PrepaidCardReceiptDataSourceFactory.java [hunk 1]
```diff
@@ -27,10 +27,12 @@
 public class PrepaidCardReceiptDataSourceFactory extends DataSource.Factory {
 
     private final MutableLiveData<PrepaidCardReceiptDataSource> mDataSourceMutableLiveData;
-    private final PrepaidCardReceiptDataSource mPrepaidCardReceiptDataSource;
+    private PrepaidCardReceiptDataSource mPrepaidCardReceiptDataSource;
+    private final String token;
```
Comment (manikandan-altmtrk) [inline receiptrepository/src/main/java/com/hyperwallet/android/ui/receipt/repository/PrepaidCardReceiptDataSourceFactory.java]: I think we are not using this variable anywhere. Am i right/

Comment (manikandan-altmtrk) [reply receiptrepository/src/main/java/com/hyperwallet/android/ui/receipt/repository/PrepaidCardReceiptDataSourceFactory.java]: Sorry, peter above comments. I think we need to use prefix to the variable.

Comment (peter-joseph) [reply receiptrepository/src/main/java/com/hyperwallet/android/ui/receipt/repository/PrepaidCardReceiptDataSourceFactory.java]: oh yeah thanks @manikandan-altmtrk 👍

File: transferui/src/main/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModel.java [hunk 2]
```diff
@@ -365,6 +381,13 @@ public void onTransferCreated(@Nullable Transfer transfer) {
             public void onError(@NonNull Errors errors) {
                 mIsLoading.postValue(Boolean.FALSE);
                 mQuoteAvailableFunds.setValue(null);
+                if (errors.containsInputError()) {
+                    Error error = errors.getErrors().get(0);
+                    if (Objects.equals(error.getFieldName(), DESTINATION_TOKEN_INPUT_FIELD)) {
+                        mInvalidDestinationError.postValue(new Event<>(error));
+                        return;
+                    }
+                }
```
Comment (jsaini-hw) [inline transferui/src/main/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModel.java]: What is the use case for this error?

**Pull request merged**